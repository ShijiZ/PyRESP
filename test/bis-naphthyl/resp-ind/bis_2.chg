All values are reported in atomic units
%FLAG TITLE
 molecule 1: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
   2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
   3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
   4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
   5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
   6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
   7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
   8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
   9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  10   7.6098920E-01   1.9305880E+00   4.8666140E-02
  11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  17   1.3078800E-02   5.9002260E+00   6.1156470E-01
  18   3.5994910E+00   2.3574250E+00   2.7072230E-02
  19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  26   8.7740430E+00   1.9007740E+00   6.2340200E-02
  27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  28   4.2425500E+00   2.6484730E+00   1.9549130E+00
  29   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6         -1    -1.0345539E-01
   2          6         -1    -5.3475632E-02
   3          6         -1    -1.5351308E-01
   4          6         -1    -4.0561954E-01
   5          6         -1    -3.8730366E-01
   6          6         -1    -1.4972817E-01
   7          6         -1    -6.4073629E-02
   8          6         -1    -3.1055385E-01
   9          6         -1    -2.9000868E-01
  10          6         -1    -8.3607568E-02
  11          1         -1     1.7371743E-01
  12          1         -1     2.6524454E-01
  13          1         -1     2.9213974E-01
  14          1         -1     2.9645324E-01
  15          1         -1     2.7912620E-01
  16          1         -1     2.6951498E-01
  17          1         -1     2.5060711E-01
  18          6          0    -5.6163728E-04
  19          6         -1     4.2979259E-01
  20          6         -1    -2.1519803E-01
  21          6         -1     3.9635359E-01
  22          8         -1    -8.7065257E-01
  23          8         -1    -8.7065257E-01
  24          1         -1    -3.0104154E-02
  25          1         -1     7.2416352E-02
  26          1         -1     4.4416147E-02
  27          1         -1     7.0997086E-02
  28          1          0     7.3864578E-02
  29          1         28     7.3864578E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   6.5322456E-03   4.2065028E-02   3.6417340E-02
   2   4.5276879E-02  -1.2690092E-02  -3.8020456E-03
   3  -2.3269863E-01   2.0318426E-01   4.1154102E-02
   4   7.7658029E-02   2.8697533E-01   3.6866015E-02
   5   3.3720979E-01  -9.5247157E-02  -2.0811069E-02
   6   1.8676521E-01  -3.7092967E-01  -5.2776665E-02
   7   7.6870205E-02   9.8103924E-02   1.3397885E-02
   8   2.3233881E-01  -2.3469504E-01  -3.6226785E-02
   9  -1.3767465E-01  -2.3222430E-01  -2.3440356E-02
  10  -1.6179738E-01   7.3150249E-03   2.2306847E-02
  11   6.6488381E-02  -2.1208086E-02   1.8088017E-02
  12  -9.1209275E-03   5.4305604E-02   1.1207555E-02
  13   4.3800181E-02   7.9829496E-02   9.9419976E-03
  14   9.4983912E-02   9.3563205E-03  -1.1193908E-03
  15   5.7672885E-02  -6.3573317E-02  -9.6321375E-03
  16   4.7801707E-02  -7.4918400E-02  -1.0634885E-02
  17  -2.2857629E-02  -7.5085873E-02  -6.6992736E-03
  18  -3.2755607E-02  -2.2684072E-01   6.5337235E-02
  19  -3.9090082E-03  -3.8773913E-01   2.5830737E-01
  20  -9.3187607E-02  -1.7311866E-01   1.6689125E-01
  21   8.4158248E-02  -6.4170388E-01   5.0048301E-01
  22   7.0884062E-03  -8.4005191E-02  -3.3511736E-03
  23   3.0926563E-02  -2.5748737E-02   8.0414436E-02
  24   2.4069374E-03  -4.7541960E-02   2.5984929E-02
  25  -1.4349339E-02  -9.4035853E-03   1.9613344E-02
  26  -1.4996585E-02  -2.9235803E-02   2.6619849E-02
  27  -1.9351103E-03  -1.4883373E-02   1.4888219E-02
  28   6.9210012E-03  -4.7220912E-02   1.6071617E-02
  29  -1.4839665E-02  -2.2289672E-02   4.6459422E-03

%FLAG TITLE
 molecule 2: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
  30  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
  31  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
  32  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
  33  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
  34  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
  35  -7.2380090E+00   1.1409360E+00   1.9545440E-01
  36  -4.5750250E+00   1.4616850E+00   1.5977450E-01
  37  -3.3997450E+00   3.8559500E+00   4.4761770E-01
  38  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  39   7.6098920E-01   1.9305880E+00   4.8666140E-02
  40   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  41  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  42  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  43  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  44  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  45  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  46   1.3078800E-02   5.9002260E+00   6.1156470E-01
  47   3.5994910E+00   2.3574250E+00   2.7072230E-02
  48   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  49   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  50   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  51   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  52   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  53   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  54   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  55   8.7740430E+00   1.9007740E+00   6.2340200E-02
  56   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  57   4.2425500E+00   2.6484730E+00   1.9549130E+00
  58   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
  30          6         -1    -1.0345539E-01
  31          6         -1    -5.3475632E-02
  32          6         -1    -1.5351308E-01
  33          6         -1    -4.0561954E-01
  34          6         -1    -3.8730366E-01
  35          6         -1    -1.4972817E-01
  36          6         -1    -6.4073629E-02
  37          6         -1    -3.1055385E-01
  38          6         -1    -2.9000868E-01
  39          6         -1    -8.3607568E-02
  40          1         -1     1.7371743E-01
  41          1         -1     2.6524454E-01
  42          1         -1     2.9213974E-01
  43          1         -1     2.9645324E-01
  44          1         -1     2.7912620E-01
  45          1         -1     2.6951498E-01
  46          1         -1     2.5060711E-01
  47          6         18    -5.6163728E-04
  48          6         -1     4.2979259E-01
  49          6         -1    -2.1519803E-01
  50          6         -1     3.9635359E-01
  51          8         -1    -8.7065257E-01
  52          8         -1    -8.7065257E-01
  53          1         -1    -3.0104154E-02
  54          1         -1     7.2416352E-02
  55          1         -1     4.4416147E-02
  56          1         -1     7.0997086E-02
  57          1         28     7.3864578E-02
  58          1         28     7.3864578E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
  30   6.5322456E-03   4.2065028E-02   3.6417340E-02
  31   4.5276879E-02  -1.2690092E-02  -3.8020456E-03
  32  -2.3269863E-01   2.0318426E-01   4.1154102E-02
  33   7.7658029E-02   2.8697533E-01   3.6866015E-02
  34   3.3720979E-01  -9.5247157E-02  -2.0811069E-02
  35   1.8676521E-01  -3.7092967E-01  -5.2776665E-02
  36   7.6870205E-02   9.8103924E-02   1.3397885E-02
  37   2.3233881E-01  -2.3469504E-01  -3.6226785E-02
  38  -1.3767465E-01  -2.3222430E-01  -2.3440356E-02
  39  -1.6179738E-01   7.3150249E-03   2.2306847E-02
  40   6.6488381E-02  -2.1208086E-02   1.8088017E-02
  41  -9.1209275E-03   5.4305604E-02   1.1207555E-02
  42   4.3800181E-02   7.9829496E-02   9.9419976E-03
  43   9.4983912E-02   9.3563205E-03  -1.1193908E-03
  44   5.7672885E-02  -6.3573317E-02  -9.6321375E-03
  45   4.7801707E-02  -7.4918400E-02  -1.0634885E-02
  46  -2.2857629E-02  -7.5085873E-02  -6.6992736E-03
  47  -3.2755607E-02  -2.2684072E-01   6.5337235E-02
  48  -3.9090082E-03  -3.8773913E-01   2.5830737E-01
  49  -9.3187607E-02  -1.7311866E-01   1.6689125E-01
  50   8.4158248E-02  -6.4170388E-01   5.0048301E-01
  51   7.0884062E-03  -8.4005191E-02  -3.3511736E-03
  52   3.0926563E-02  -2.5748737E-02   8.0414436E-02
  53   2.4069374E-03  -4.7541960E-02   2.5984929E-02
  54  -1.4349339E-02  -9.4035853E-03   1.9613344E-02
  55  -1.4996585E-02  -2.9235803E-02   2.6619849E-02
  56  -1.9351103E-03  -1.4883373E-02   1.4888219E-02
  57   6.9210012E-03  -4.7220912E-02   1.6071617E-02
  58  -1.4839665E-02  -2.2289672E-02   4.6459422E-03

