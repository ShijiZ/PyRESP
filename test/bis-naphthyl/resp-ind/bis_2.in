test for bis-naphthyl
 &cntrl 
 nmol = 2,
 iqopt = 2,
 ihfree = 1,
 qwt = 0.001,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 0,
 &end 
    1.0
naphthyl
   -1   29
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1 
    6   -1 
    6   -1   
    6   -1 
    1   -1 
    1   -1 
    1   -1 
    1   -1 
    1   -1   
    1   -1 
    1   -1 
    6    0 
    6   -1 
    6   -1 
    6   -1   
    8   -1 
    8   -1 
    1   -1 
    1   -1 
    1   -1  
    1   -1 
    1    0 
    1   28 

    1.0
naphthyl
   -1   29
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1 
    6   -1 
    6   -1   
    6   -1 
    1   -1 
    1   -1 
    1   -1 
    1   -1 
    1   -1   
    1   -1 
    1   -1 
    6    0 
    6   -1 
    6   -1 
    6   -1   
    8   -1 
    8   -1 
    1   -1 
    1   -1 
    1   -1  
    1   -1 
    1    0 
    1   28 


    2
    1   18    2   18
    2
    1   28    2   28





