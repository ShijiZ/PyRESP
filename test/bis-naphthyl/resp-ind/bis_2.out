
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for bis-naphthyl
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 2   iqopt       = 2
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00100000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0

 --------------------
 Molecule Information
 --------------------
 Multiple-molecule run of 2 molecules

 Molecule 1 weight:     1.000
 Molecule 1 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
    1    6   -1
    2    6   -1
    3    6   -1
    4    6   -1
    5    6   -1
    6    6   -1
    7    6   -1
    8    6   -1
    9    6   -1
   10    6   -1
   11    1   -1
   12    1   -1
   13    1   -1
   14    1   -1
   15    1   -1
   16    1   -1
   17    1   -1
   18    6    0
   19    6   -1
   20    6   -1
   21    6   -1
   22    8   -1
   23    8   -1
   24    1   -1
   25    1   -1
   26    1   -1
   27    1   -1
   28    1    0
   29    1   28

 Molecule 2 weight:     1.000
 Molecule 2 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
   30    6   -1
   31    6   -1
   32    6   -1
   33    6   -1
   34    6   -1
   35    6   -1
   36    6   -1
   37    6   -1
   38    6   -1
   39    6   -1
   40    1   -1
   41    1   -1
   42    1   -1
   43    1   -1
   44    1   -1
   45    1   -1
   46    1   -1
   47    6    0
   48    6   -1
   49    6   -1
   50    6   -1
   51    8   -1
   52    8   -1
   53    1   -1
   54    1   -1
   55    1   -1
   56    1   -1
   57    1    0
   58    1   28

 -------------------------------
 Charges Equivalence Information 
 -------------------------------
    1   18    2   18
    1   28    2   28

 -----------------------------
 Atom Ivary Information
 -----------------------------
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    6    0
    6   -1
    6   -1
    6   -1
    8   -1
    8   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1    0
    1   28

    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    6   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   -1
    6   18
    6   -1
    6   -1
    6   -1
    8   -1
    8   -1
    1   -1
    1   -1
    1   -1
    1   -1
    1   28
    1   28

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   58
 Weight factor on initial charge restraints=  0.001000
 There are  2 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 Reading esp's for molecule   2
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 ------------------------
 Optimization Information
 ------------------------
 Since iqopt>1,   58 new q0 values will be read from file
 bis_1.chg

 Number of unique UNfrozen charge centers =     2

 Non-linear optimization requested.
  qchnge =    4.2099601129E-02
  qchnge =    4.3772955199E-06
  qchnge =    2.6466135988E-11

 Convergence in    2 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: naphthyl
 molecule 2: naphthyl

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6       -0.103455     -0.103455     -1       0.006950
    2         6       -0.053476     -0.053476     -1       0.008818
    3         6       -0.153513     -0.153513     -1       0.005458
    4         6       -0.405620     -0.405620     -1       0.002394
    5         6       -0.387304     -0.387304     -1       0.002500
    6         6       -0.149728     -0.149728     -1       0.005554
    7         6       -0.064074     -0.064074     -1       0.008420
    8         6       -0.310554     -0.310554     -1       0.003065
    9         6       -0.290009     -0.290009     -1       0.003260
   10         6       -0.083608     -0.083608     -1       0.007672
   11         1        0.173717      0.173717     -1       0.000000
   12         1        0.265245      0.265245     -1       0.000000
   13         1        0.292140      0.292140     -1       0.000000
   14         1        0.296453      0.296453     -1       0.000000
   15         1        0.279126      0.279126     -1       0.000000
   16         1        0.269515      0.269515     -1       0.000000
   17         1        0.250607      0.250607     -1       0.000000
   18         6       -0.002271     -0.000562      0       0.010000
   19         6        0.429793      0.429793     -1       0.002266
   20         6       -0.215198     -0.215198     -1       0.004214
   21         6        0.396354      0.396354     -1       0.002446
   22         8       -0.870653     -0.870653     -1       0.001141
   23         8       -0.870653     -0.870653     -1       0.001141
   24         1       -0.030104     -0.030104     -1       0.000000
   25         1        0.072416      0.072416     -1       0.000000
   26         1        0.044416      0.044416     -1       0.000000
   27         1        0.070997      0.070997     -1       0.000000
   28         1        0.058334      0.073865      0       0.000000
   29         1        0.091104      0.073865     28       0.000000

   30         6       -0.103455     -0.103455     -1       0.006950
   31         6       -0.053476     -0.053476     -1       0.008818
   32         6       -0.153513     -0.153513     -1       0.005458
   33         6       -0.405620     -0.405620     -1       0.002394
   34         6       -0.387304     -0.387304     -1       0.002500
   35         6       -0.149728     -0.149728     -1       0.005554
   36         6       -0.064074     -0.064074     -1       0.008420
   37         6       -0.310554     -0.310554     -1       0.003065
   38         6       -0.290009     -0.290009     -1       0.003260
   39         6       -0.083608     -0.083608     -1       0.007672
   40         1        0.173717      0.173717     -1       0.000000
   41         1        0.265245      0.265245     -1       0.000000
   42         1        0.292140      0.292140     -1       0.000000
   43         1        0.296453      0.296453     -1       0.000000
   44         1        0.279126      0.279126     -1       0.000000
   45         1        0.269515      0.269515     -1       0.000000
   46         1        0.250607      0.250607     -1       0.000000
   47         6       -0.002271     -0.000562     18       0.010000
   48         6        0.429793      0.429793     -1       0.002266
   49         6       -0.215198     -0.215198     -1       0.004214
   50         6        0.396354      0.396354     -1       0.002446
   51         8       -0.870653     -0.870653     -1       0.001141
   52         8       -0.870653     -0.870653     -1       0.001141
   53         1       -0.030104     -0.030104     -1       0.000000
   54         1        0.072416      0.072416     -1       0.000000
   55         1        0.044416      0.044416     -1       0.000000
   56         1        0.070997      0.070997     -1       0.000000
   57         1        0.058334      0.073865     28       0.000000
   58         1        0.091104      0.073865     28       0.000000

 Sum over the calculated charges:     -2.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          3.2877896E+01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   9.3391958E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.9327903E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.6853990E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1       12.86340  -10.93276    6.15456   -2.83956
 #MOL         D          Dx         Dy         Dz
   2       12.86340  -10.93276    6.15456   -2.83956

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X  -21.08616
        Y   17.61284   10.89822
        Z  -11.55289    4.39465   10.18794
 #MOL          X          Y          Z
   2    X  -21.08616
        Y   17.61284   10.89822
        Z  -11.55289    4.39465   10.18794

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   18.77642
        Y    0.00000   13.68070
        Z    0.00000    0.00000  -32.45712
 #MOL          X          Y          Z
   2    X   18.77642
        Y    0.00000   13.68070
        Z    0.00000    0.00000  -32.45712